Dockerfile

```diff
@@ -337,12 +339,13 @@ WORKDIR /go/src/github.com/opencontainers/runc
 ARG DEBIAN_FRONTEND
 ARG TARGETPLATFORM
 RUN xx-apt-get install -y --no-install-recommends dpkg-dev gcc libc6-dev libseccomp-dev
+ARG DOCKER_STATIC
 RUN --mount=from=runc-src,src=/usr/src/runc,rw \
     --mount=type=cache,target=/root/.cache <<EOT
   set -e
   xx-go --wrap
-  CGO_ENABLED=1 make BUILDTAGS="seccomp" runc
-  xx-verify runc
+  CGO_ENABLED=1 make BUILDTAGS="seccomp" "$([ "$DOCKER_STATIC" = "1" ] && echo "static" || echo "runc")"
```

Comment:
I think we should be able to drop the `seccomp` build-tag; it should be enabled by default; https://github.com/opencontainers/runc/tree/v1.1.4#build-tags